.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+    // See http://go.microsoft.com/fwlink/?LinkId=827846
```

Comment:
I didn't think that comments were allowed in JSON

Reply:
Well, per JSON standard, you are right but it's no problem for VSCode, similar comments are also in the .vscode folder of the PowerShell Core repo

Reply:
ok!